buildenv/jenkins/common/variables-functions

```diff
@@ -207,6 +216,17 @@ def set_test_variables() {
     TEST_DEPENDENCIES_JOB_NAME = VARIABLES.test_dependencies_job_name
 }
 
+/*
+* Set TESTS_TARGETS, indicating the level of testing. 
+*/
+def set_test_targets() {
+    TESTS_TARGETS = params.TESTS_TARGETS
+    if ((TESTS_TARGETS == null) || (TESTS_TARGETS == '')) {
+        // set default TESTS_TARGETS for pipeline job (run all tests)
```

Comment:
presume by looking at default value, that parameter accepts a comma-separated list of test targets, any restrictions?

Reply:
Yes, it is restricted to _sanity and _extended for now.
We'll add a generic pipeline to be able to run any tests.